pkg/tenant/db/organizations.go

```diff
@@ -0,0 +1,72 @@
+package db
+
+import (
+	"context"
+
+	"github.com/oklog/ulid/v2"
+	"github.com/rotationalio/ensign/pkg/utils/ulids"
+)
+
+const OrganizationNamespace = "organizations"
+
+// Use the resourceID to retrieve the orgID from the database.
+func GetOrgIndex(ctx context.Context, resourceID ulid.ULID) (orgID ulid.ULID, err error) {
+	if ulids.IsZero(resourceID) {
+		return ulid.ULID{}, ErrMissingID
+	}
+
+	var data []byte
+	if data, err = getRequest(ctx, OrganizationNamespace, resourceID[:]); err != nil {
+		return ulid.ULID{}, err
+	}
+
+	if err = orgID.UnmarshalBinary(data); err != nil {
+		return ulid.ULID{}, err
+	}
+	return orgID, nil
+}
+
+// Store the resourceID as a key and the orgID as a value in the database.
+func PutOrgIndex(ctx context.Context, resourceID, orgID ulid.ULID) error {
+	if ulids.IsZero(resourceID) {
+		return ErrMissingID
+	}
+
+	if ulids.IsZero(orgID) {
+		return ErrMissingOrgID
+	}
+
+	if err := putRequest(ctx, OrganizationNamespace, resourceID[:], orgID[:]); err != nil {
+		return err
+	}
```

Comment:
Can we create a story for the following TODO please?

```suggestion
	// TODO: before doing the put request perform an existence check to make sure the key is not being overwritten
	if err := putRequest(ctx, OrganizationNamespace, resourceID[:], orgID[:]); err != nil {
		return err
	}
```